src/main/java/bio/terra/app/usermetrics/BardClient.java

```diff
@@ -24,29 +30,43 @@ public class BardClient {
   private final UserMetricsConfiguration metricsConfig;
 
   private final RestTemplate restTemplate;
+
   private final HttpHeaders headers;
 
+  private static final int DEFAULT_BEARER_TOKEN_CACHE_TIMEOUT_SECONDS = 3600;
+
+  private final Map<String, String> bearerCache;
+
   @Autowired
   public BardClient(UserMetricsConfiguration metricsConfig) {
     this.restTemplate = new RestTemplate();
     restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory());
+
     this.headers = new HttpHeaders();
     headers.setContentType(MediaType.APPLICATION_JSON);
     headers.setAccept(Arrays.asList(MediaType.APPLICATION_JSON, MediaType.APPLICATION_JSON));
 
+    int ttl =
+        Objects.requireNonNullElse(
+            metricsConfig.getSyncRefreshIntervalSeconds(),
+            DEFAULT_BEARER_TOKEN_CACHE_TIMEOUT_SECONDS);
```

Comment:
I wonder if this defaulting should be implemented in the config class or here. In theory, if more than one class was using the config, you might want them all to have the same default, and avoid the `null` check in more than one place. So the question to me is, is this a global config default or a per-class default for this config property?